app/uk/gov/hmrc/apigatekeeperapisfrontend/controllers/ApiListController.scala

```diff
@@ -38,9 +42,72 @@ class ApiListController @Inject() (
 
   val page: Action[AnyContent] = loggedInOnly() { implicit request =>
     apmService.fetchAllApis()
+      .map(_.distinct)
       .map(_.sortBy(_.name.toLowerCase))
       .map(nameContext => Ok(apiListPage(nameContext)))
 
   }
 
+  private def isTrial(apiVersion: ApiVersion): Boolean = {
+    apiVersion.access match {
+      case ApiAccess.Private(true) => true
+      case _                       => false
+    }
+  }
+
+  val csv: Action[AnyContent] = loggedInOnly() { implicit request =>
+    val columnDefinitions: Seq[ColumnDefinition[ApiDefinitionView]] = Seq(
+      ColumnDefinition("name", vm => vm.name),
```

Comment:
What does `vm` mean?

Reply:
not sure, just an interim variable name which I copied from the CSV code from api-gatekeeper-frontend